modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/OrthologAnnotation_conf.pm

```diff
@@ -0,0 +1,185 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 NAME
+
+ Bio::EnsEMBL::Production::Pipeline::PipeConfig::OrthologAnnotation_conf;
+
+=head1 DESCRIPTION
+
+=head1 AUTHOR 
+
+ mchakiachvili@ebi.ac.uk
+
+=cut
+
+package Bio::EnsEMBL::Production::Pipeline::PipeConfig::OrthologAnnotation_conf;
```

Comment:
This seems like a confusing name for a pipeline that is designed to dump a list of species for which we want GOA annotations - by definition it has nothing to do with orthologs.
```suggestion
package Bio::EnsEMBL::Production::Pipeline::PipeConfig::DumpSpeciesForGOA_conf;
```